community/php7/php7-fpm.initd

```diff
@@ -6,8 +6,6 @@
 # variable "user" and optionally "group" in conf.d file, or in the $fpm_config
 # file (the former has precedence).
 
-: ${name:="PHP FastCGI Process Manager"}
```

Comment:
But I think it's better to use name of init-script ($RC_SVCNAME) as unique identifier in logs